crates/workspace/src/workspace.rs

```diff
@@ -5728,20 +5728,29 @@ impl Render for Workspace {
         let centered_layout = self.centered_layout
             && self.center.panes().len() == 1
             && self.active_item(cx).is_some();
+
+        let centered_settings = WorkspaceSettings::get_global(cx).centered_layout;
+        let show_border = centered_layout && centered_settings.show_border;
+
         let render_padding = |size| {
             (size > 0.0).then(|| {
-                div()
+                let el = div()
                     .h_full()
                     .w(relative(size))
-                    .bg(cx.theme().colors().editor_background)
-                    .border_color(cx.theme().colors().pane_group_border)
+                    .bg(cx.theme().colors().editor_background);
+
+                if show_border {
```

Comment:
`gpui` has `.when(show_border, |el| ` method, let's preserve the builder notation here and in other places below.